app/nagbot.py

```diff
@@ -36,86 +30,67 @@
-
-        instances_to_terminate = get_terminatable_instances(instances)
-        if len(instances_to_terminate) > 0:
-            terminate_msg = 'The following %d _stopped_ instances are due to be *TERMINATED*, ' \
-                            'based on the "Terminate after" tag:\n' % len(instances_to_terminate)
-            for i in instances_to_terminate:
-                contact = sqslack.lookup_user_by_email(i.contact)
-                terminate_msg += make_instance_summary(i) + ', "Terminate after"={}, "Monthly Price"={}, Contact={}\n' \
-                    .format(i.terminate_after, money_to_string(i.monthly_price), contact)
-                sqaws.set_tag(i.region_name, 'instance', i.instance_id, i.terminate_after_tag_name,
-                              parsing.add_warning_to_tag(i.terminate_after, TODAY_YYYY_MM_DD), dryrun=dryrun)
-        else:
-            terminate_msg = 'No instances are due to be terminated at this time.\n'
-        sqslack.send_message(channel, terminate_msg)
-
-        instances_to_stop = get_stoppable_instances(instances)
-        if len(instances_to_stop) > 0:
-            stop_msg = 'The following %d _running_ instances are due to be *STOPPED*, ' \
-                       'based on the "Stop after" tag:\n' % len(instances_to_stop)
-            for i in instances_to_stop:
-                contact = sqslack.lookup_user_by_email(i.contact)
-                stop_msg += make_instance_summary(i) + ', "Stop after"={}, "Monthly Price"={}, Contact={}\n' \
-                    .format(i.stop_after, money_to_string(i.monthly_price), contact)
-                sqaws.set_tag(i.region_name, 'instance', i.instance_id, i.stop_after_tag_name,
-                              parsing.add_warning_to_tag(i.stop_after, TODAY_YYYY_MM_DD, replace=True), dryrun=dryrun)
-        else:
-            stop_msg = 'No instances are due to be stopped at this time.\n'
-        sqslack.send_message(channel, stop_msg)
-
-        volumes_to_delete = get_deletable_volumes(volumes)
-        if len(volumes_to_delete) > 0:
-            delete_msg = 'The following %d volumes are due to be *DELETED*, ' \
-                         'based on the "Terminate after" tag:\n' % len(volumes_to_delete)
-            for v in volumes_to_delete:
-                contact = sqslack.lookup_user_by_email(v.contact)
-                delete_msg += make_volume_summary(v) + ', "Terminate after"={}, "Monthly Price"={}, Contact={}\n' \
-                    .format(v.terminate_after, money_to_string(v.monthly_price), contact)
-                sqaws.set_tag(v.region_name, 'volume', v.volume_id, v.terminate_after_tag_name,
-                              parsing.add_warning_to_tag(v.terminate_after, TODAY_YYYY_MM_DD, replace=True),
-                              dryrun=dryrun)
-        else:
-            delete_msg = 'No volumes are due to be deleted at this time.\n'
-        sqslack.send_message(channel, delete_msg)
+        sqslack.send_message(channel, terminate_message)
```

Comment:
That's right, I removed that code for now because Cody Ray said he wasn't sure how helpful it actually was to do and as it was it would have needed tweaking to get it to work with the changes.